chain-impl-mockchain/src/block.rs

```diff
@@ -13,7 +14,17 @@ pub struct Block {
     pub slot_id: BlockDate,
     pub parent_hash: Hash,
 
-    pub transactions: Vec<SignedTransaction>,
+    pub messages: Vec<Message>,
```

Comment:
`block_contents: Vec<Message>` I think we want to highlight that this si the content of the block and everything else is meta.